### ISSUE TITLE
Strange Tearing with some DXVK Games

### ISSUE THREAD

**Issue by Alexmitter** [doitsujin/dxvk#681]
With some games running on DXVK, there is a breaking line in the drawn image, like the top part gets send to the display one frame behind. The line is stable and does not move a lot like a tearing line normally does. In terms of some minutes, it moves slightly up or down. Framerate is quite stable around 60fps.
Video: https://youtu.be/RtnVlt6pglc

### Software information
GTA5, Direct X 10 and 11. Vsync on or off. From lowest to highest settings.
Skyrim, also settings doesn't change the behavior.

### System information
- GPU: GTX 1060
- Driver: 396.54
- Wine version: Proton 3.7.7 and 3.7.6
- DXVK version: Proton Bundled Version

**Comment by doitsujin** [comment]
Not my bug, this is just what currently happens on Nvidia, regardless of whether you enable Vsync or not.

**Comment by setsunati** [comment]
@Alexmitter Try enabling force composition pipeline & force full composition pipeline in nvidia-settings.

https://www.youtube.com/watch?v=OqrtvIl0Gcw

**Comment by pchome** [comment]
@Alexmitter 
> The nvidia settings require me to press on every monitor, select composition pipeline and press apply. 

`$ cat /etc/X11/xorg.conf.d/70-device.conf | grep -i pipeline`
```
    # for >= 375.10 and ForceFullCompositionPipeline
    Option  "ForceCompositionPipeline" "true"
    # fix CompositionPipeline GPU usage for >= 390.25
    Option "UseNvKmsCompositionPipeline" "false"
#    Option  "ForceFullCompositionPipeline" "true"
```
